api/src/OpenTelemetry/Internal/Trace/Id.hs

```diff
@@ -52,6 +52,7 @@ import GHC.Exts (
   int2Word#,
   isTrue#,
   or#,
+  word64ToWord#,
```

Comment:
It seems that base older than 4.17.0.0 does not expose `word64Word#`.

Please use CPP conditional branch.